llm/app/core/settings/user_reservation_bot.yaml

```
@@ -0,0 +1,149 @@
+
+      ## 출력 형식 (반드시 이 형식만):
+      {{
+          "intent": "view_reservations|cancel_reservation|search_shops|unknown"
+      }}
+
+      ---
+      이제, 다음 사용자 요청을 분석하여 규칙에 따라 JSON 객체 하나만 반환하세요.
+      ---
+      사용자 요청: {query}
+    default_message: "현재 요청을 처리할 수 없습니다. 잠시 후 다시 시도해주세요."
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)